log/logger.go

```diff
@@ -358,6 +366,23 @@ func (s handlerSpawner) NeedsSpawn(attributes map[AttributeKey]string) bool {
 	return true
 }
 
+// createFileHandlerForParent creates a JSON handler for logging into a file.
+// REMARK The returned handler may also delegate to any "plainTextHandler", that the parent is already delegating to.
```

Comment:
we only delegate to `plainTextHandler` because STDOUT is such a handler, and because all other handlers are not. is that correct?
if we would delegate to all other handlers as well, we would have the same log file tree structure as before, right?